docs/sdk-and-tools/sdk-js/sdk-js-signing-providers.md

```diff
@@ -11,36 +11,65 @@ This page will guide you through the process of integrating the **sdk-js signing
 Note that for most purposes, **we recommend using [sdk-dapp](https://github.com/multiversx/mx-sdk-dapp)** instead of integrating the signing providers on your own.
 :::
 
-:::important
-The code samples depicted on this page can also be found on the [**sdk-js examples repository**](https://github.com/multiversx/mx-sdk-js-examples).
-:::
+[comment]: # (mx-context-auto)
+
+## Anatomy of a signing provider
+
+Generally speaking, a signing provider is a component that supports the following use-cases:
+
+ - **Login (trivial flow, not recommended)**: the user of a dApp is asked her MultiversX identity. The user reaches the wallet, unlocks it, and confirms the login. The flow continues back to the dApp, which is now informed about the user's blockchain address. Note, though, that this piece of information is not authenticated: the dApp receives a _hint_ about the user's address, not a _guarantee_ (proof). Sometimes (though rarely), this is enough. If in doubt, always have your users login using the **native authentication** flow (see below).
```

Comment:
login->log in and logout->log out where needed
login/logout are nouns (for example: the login page)
log in/log out are actions (for example: please log in)